hw3/01-characters.js

```diff
@@ -1,4 +1,57 @@
 // url for the Thrones API
-const url = 'https://thronesapi.com/api/v2/Characters';
+const url = "https://thronesapi.com/api/v2/Characters";
 
-fetch(url);
+// fetch(url);
+
+//get the user data and render the data inside the <div> element with the class container
+// let container = document.querySelector(".container");
+
+//reference --https://www.javascripttutorial.net/javascript-fetch-api/
+
+let content = document.querySelector("#content");
+
+function show(user) {
```

Comment:
I'm not sure `user` is an appropriate identifier here. Since we're retrieving and displaying character information, `character` seems more accurate and descriptive.